sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -671,7 +671,12 @@ public void setProxy(@Nullable Proxy proxy) {
    * @param sampleRate the sample rate
    */
   public void setSampleRate(Double sampleRate) {
-    this.validateRate(sampleRate);
+    if (sampleRate != null && (sampleRate > 1.0 || sampleRate <= 0.0)) {
+      throw new IllegalArgumentException(
+          "The value "
+              + sampleRate
+              + " is not valid. Use null to disable or values between 0.01 and 1.0.");
```

Comment:
```suggestion
              + " is not valid. Use null to disable or values > 0.0 and <= 1.0.");
```

the error message is misleading, 0.001 would be valid already but the log says 0.01